### ISSUE TITLE
2.5.0: JPA modelgen classes are breaking the Gradle plugin

### ISSUE THREAD

**Issue by juergenzimmermann** [stoicflame/enunciate#421]
I just tried to upgrade from 2.2.0 to 2.5.0. However, I'm using JPA's criteria queries and the generated metamodel classes. Via https://plugins.gradle.org/plugin/at.comm_unity.gradle.plugins.jpamodelgen these classes are generated in src/jpaModelgen, so that there is a second source directory in addition to src/main.
Using Enunciate 2.2 there was just a kind of warning and the build continued. This behaviour is ok since these classes are not required for the REST API (only for the database access via JPA). Now the build stops with an error message like this one:
```
[javac] /C:/Users/ziju0001/Documents/NetBeansProjects/shop/src/main/java/de/shop/bestellverwaltung/business/WarenkorbRemover.java:21: error: cannot find symbol
[javac] import de.shop.bestellverwaltung.domain.Warenkorbposition_;
[javac]                                        ^
[javac]   symbol:   class Warenkorbposition_
[javac]   location: package de.shop.bestellverwaltung.domain
:enunciate FAILED
```

**Comment by stoicflame** [comment]
I'm actually surprised it wasn't failing for you before.

We need some way to add those generated classes to the Enunciate source path, but it doesn't look like there's currently a way to do that. I've opened stoicflame/enunciate-grade#9 to track that work.

**Comment by stoicflame** [comment]
Fixed with stoicflame/enunciate-gradle#9 and #427.
